conda_content_trust/plugin.py

```diff
@@ -1,13 +1,23 @@
 # Copyright (C) 2019 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
-import conda.plugins
+
+from conda import __version__ as conda_version
 
 from .cli import cli
 
+version_parts = conda_version.split('.')
+major = int(version_parts[0]) # year
+minor = int(version_parts[1]) if len(version_parts) > 1 else 0 # month
+
+if (major, minor) >= (25, 9):
+    from conda.plugins.types import CondaSubcommand
+else:
+    from conda.plugins import CondaSubcommand
+
```

Comment:
Do we need the version handling here that I introduced in 7cfb2edaf10e413609c8d91dd0bffaeef6ddf41c, or will just using the new API via 31ef887d2eb9600cd58a0c6d7ee98a288587c797 suffice?

Reply:
Neat, thanks!